docs/semgrep-cloud-platform/asana.md

```diff
@@ -0,0 +1,93 @@
+
+:::tip Try our Asana integration (beta)
+* Existing customers can try our Asana integration for free. To enable this feature:
+    1. Fill out the following form: [Request access to the Semgrep Asana integration private beta](https://get.semgrep.dev/Jira-private-beta.html).
+    2. Contact your Technical Account Manager or your Account Executive and let them know you'd like to try the Asana integration out.
+:::
+
+The Semgrep Asana integration allows you to create Asana tickets based on your Semgrep Code or Supply Chain findings.
+
+## Enabling your Asana integration
+
+:::info Prerequisites
+* Your Asana user must be able to approve an OAuth integration with admin level privileges.
+* Asana can only be enabled through Semgrep Cloud Platform (SCP). [Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to set up your Asana integration.
+* To create tickets, you must [add or onboard the project](/semgrep-code/getting-started/#option-b-adding-a-repository-from-github-gitlab-or-bitbucket) (repository) to Semgrep Cloud Platform for scanning.
+:::
+
+To integrate Asana, follow these steps:
+
+1. In [Semgrep Cloud Platform](https://semgrep.dev/login), go to **Settings** > **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)**.
+2. On the **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)** page, click **Add Integration** (or **Setup First Integration** if this is your first integration), and then select **Ticketing > Asana**.
+3. Follow the on-screen instructions to integrate Asana.
+
+## Configuring your default project
+
+Use the drop-down to select the default project. These settings can be changed later from the integrations page.
+
+![Asana configuration modal](/img/asana-configure-defaults.png)
+
+**Figure** A sample Asana integration
+
+## Creating tickets
+
+### Code
+
+:::note
+Creating tickets for many findings at once may take some time. Tickets that take longer than 10 seconds to create will be shown in the UI once the page is refreshed.
+:::
+
+To create tickets for one or more Code findings, use the triage button from the Code findings page or from an individual finding's details page. Findings can be individually selected, or you can create tickets for all findings for a given rule. 
+
+Once a ticket has been created, a link will be present on the right side of the findings page and along the top of an individual finding's details page.
+
+![Create Asana ticket - Code](/img/asana-code-findings.png)
+**Figure** Code triage flow
+
+![Asana ticket created - Code](/img/asana-code-ticketed.png)
+**Figure** Code ticket created
+
+### Supply Chain
+
+:::caution Limitations
+Tickets can only be created for Supply Chain findings with reachable usages. They cannot be created for **Always reachable** or **Unreachable** findings.
+:::
+
+The Supply Chain triage flow will create tickets for all instances of a given vulnerability. Instances can not be selected individually. 
```

Comment:
```suggestion
The Supply Chain triage flow creates tickets for all instances of a given vulnerability. Instances can not be selected individually. 
```

Curious - is there a difference between "instance" and "usage"? If they are synonymous, prefer "usage" so that the user isn't confused.